packages/web-components/src/components/va-checkbox/va-checkbox.tsx

```diff
@@ -188,11 +191,15 @@ export class VaCheckbox {
         'usa-legend': true,
         'usa-label--error': error
       });
-      const ariaDescribedbyIds = `${messageAriaDescribedby ? 'input-message' : ''} ${error ? 'checkbox-error-message' : ''}`
-      .trim() || null; // Null so we don't add the attribute if we have an empty string
+      const ariaDescribedbyIds = [
+        messageAriaDescribedby ? 'input-message' : '',
+        error ? 'checkbox-error-message' : '',
+        hasDescription ? 'description' : '',
```

Comment:
Would there ever be a scenario when we wouldn't want the description content associated with aria-describedby?

Reply:
I think if no description is provided?

Reply:
Oh, I see what you're saying... Monday was really cray-cray for me. I can't think of any scenario like that. If it ever came up, we could modify the code to accommodate the need.